packages/js/product-editor/src/blocks/description/edit.tsx

```diff
@@ -45,7 +55,11 @@ export function Edit() {
 				<ModalEditor
 					initialBlocks={ parse( description ) }
 					onChange={ ( blocks ) => {
-						const html = serialize( blocks );
+						// By default the blocks variable always contains one paragraph
+						// block with empty content, that causes the desciption to never
+						// be empty. The next line removes the default block to keep
+						// the description empty.
```

Comment:
I think that if you rename the function as suggested above, you can also move this comment inside the function, as it isn't needed here with a more specific function name and the comment really helps to better understand the function implementation.

Reply:
done -> [4373e02](https://github.com/woocommerce/woocommerce/pull/39229/commits/4373e021566b3a394ce8087309c133a41a4ef69d)